testing/sdk_utils.py

```diff
@@ -243,3 +242,61 @@ def merge_dictionaries(dict1: Dict, dict2: Optional[Dict]) -> Dict:
         else:
             ret[k] = dict2[k]
     return ret
+
+
+def get_service_roles(service_name) -> dict:
+
+    # Get the current service state to verify roles have applied.
+    mesos_state = sdk_cmd.cluster_request("GET", "/mesos/master/state").json()
+
+    service_state = None
+    current_service_roles = {}
+
+    # Find our service.
+    for service in mesos_state["frameworks"]:
+        if service["name"] == service_name:
+            service_state = service
+
+    # Create a map of tasks to roles.
+    if service_state:
+
+        if "roles" in service_state:
+            # MUTI_ROLE
+            current_service_roles["framework-roles"] = service_state["roles"]
```

Comment:
So if this is a multi role, what would be the result of `service_state["role"]` - is it a non existent key OR defaulted to an empty list ?

Reply:
`service_state["role"] = "*"`